src/bundles/mark_sweep/index.ts

```diff
@@ -184,34 +128,16 @@ function newCommand(
 }
 
 function newSweep(left, heap): void {
-  const toSpace = 0;
-  const fromSpace = 0;
   const newSizeLeft = NODE_SIZE;
   const desc = `Freeing node ${left}`;
-  newCommand(
-    'SWEEP',
-    toSpace,
-    fromSpace,
-    left,
-    -1,
-    newSizeLeft,
-    0,
-    heap,
-    desc,
-    'freed node',
-    ''
-  );
+  newCommand('SWEEP', left, -1, newSizeLeft, 0, heap, desc, 'freed node', '');
 }
 
 function newMark(left, heap, queue): void {
-  const toSpace = 0;
-  const fromSpace = 0;
   const newSizeLeft = NODE_SIZE;
   const desc = `Marking node ${left} to be live memory`;
   newCommand(
     'MARK',

```

Comment:
Try not to add magic strings here unless it's absolutely necessary. Again, you can consider using TypeScript's enum.